app/mailers/user_mentee_application_alert_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class UserMenteeApplicationAlertMailer < ApplicationMailer
+  def notify_for_application_submission
+    @user_mentee_application = params[:user_mentee_application]
+    @user = User.admin.first
+    mail(to: @user.email, subject: 'Mentee Application Submitted')
```

Comment:
So a couple of things here:

If you look in `ApplicationMailer`, you'll see that mailers inherit a `set_recipient` method and a `before_action` filter that calls it. Noticed classes automatically pass the mailer the `recipient` param for whoever the notification is being delivered to.

`ApplicationMailer` also specifies that the default `to` is for the recipient's email address. This all basically just allows us to write these emails with less repetition. This method could just become
```rb
def notify_for_application_submission
  @user_mentee_application = params[:user_mentee_application]
  mail(subject: 'Mentee Application Submitted')
end
```
And then in the template, you could refer to `@user_mentee_application` if you need data from that (which you're already doing), but you'll also have `@recipient` to refer to the user that is receiving the email.

And same stuff goes for the other mailer. Definitely let me know if any of this is confusing.

Reply:
Oh that's awesome! changes made!